src/improvements/template/OPCMStage1.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
What is this Template? Which version of OPCM? Can you follow the OPCM template pattern we have already e.g. OPCMUpgradeV300.sol.